src/sentry/monitors/endpoints/organization_monitor_index.py

```diff
@@ -259,10 +259,16 @@ def post(self, request: Request, organization) -> Response:
 
         result = validator.validated_data
 
+        if result.get("owner"):
+            owner_id = result.get("owner").id
+        else:
+            owner_id = None
```

Comment:
Don't really like having to write it like this.

If we were using a `FlexibleForeignKey` on the `owner_actor` column django would handle accepting an Actor. But we can't do that here since it's on a different db.